sources/packages/web/src/services/http/dto/Restriction.dto.ts

```diff
@@ -45,6 +45,16 @@ export interface AssignRestrictionAPIInDTO extends ResolveRestrictionAPIInDTO {
   restrictionId: number;
 }
 
+export interface AssignRestrictionCategoryOutDTO {
```

Comment:
These ones are not following API naming convention, is it a gap from the API reorganization?